src/api/services/store/storeService.test.ts

```diff
@@ -0,0 +1,206 @@
+import { test, expect, beforeEach, vi } from "vitest";
+import { createStoreService } from "./storeService";
+import { mockConfig, mockMembers } from "../common/fixtures";
+import { Grades } from "../../type";
+
+// Mock services
+const mockFetchMembers = vi.fn();
+const mockFetchSubmissions = vi.fn();
+const mockAnalyzeMemberInfo = vi.fn();
+
+// Mock localStorage
+const mockLocalStorage = {
+  getItem: vi.fn(),
+  setItem: vi.fn(),
+  removeItem: vi.fn(),
+};
+Object.defineProperty(window, "localStorage", { value: mockLocalStorage });
+
+vi.mock("../fetch/fetchService", () => ({
+  createFetchService: () => ({
+    fetchMembers: mockFetchMembers,
+    fetchSubmissions: mockFetchSubmissions,
+  }),
+}));
```

Comment:
`vi.mock()`을 이런 식으로 사용하시면 모킹하는 모듈의 타입을 완전히 무시해버리기 때문에 나중에 `fetchService`의 API가 바뀌었을 때도 테스트가 통과하는 문제가 발생하게 됩니다. 다음과 같이 Type Saftety를 잃어버리지 않도록 테스트를 작성해주시면 좋을 것 같습니다.

```suggestion
import { createFetchService } from "../fetch/fetchService";

vi.mock("../fetch/fetchService");

vi.mocked(createFetchService).mockReturnValue({
  fetchMembers: mockFetchMembers,
  fetchSubmissions: mockFetchSubmissions,
});
```